### ISSUE TITLE
The kubernetes server version info is wrong  for minikube v0.25.0

### ISSUE THREAD

**Issue by jdxyw** [kubernetes/minikube#2593]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

This bug has been reported and closed in this issue before in v0.19.0. https://github.com/kubernetes/minikube/issues/1532

Please provide the following details:

**Environment**:

**Minikube version** (use `minikube version`): minikube version: v0.25.0
- **OS** (e.g. from /etc/os-release):mac os x 16.7.0 Darwin
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): "DriverName": "virtualbox"
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): minikube-v0.25.1.iso
- **Install tools**: 
- **Others**:
The above can be generated in one go with the following commands (can be copied and pasted directly into your terminal):

**What happened**:

**What you expected to happen**:
The server version info is empty
```
Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.4", GitCommit:"793658f2d7ca7f064d2bdf606519f9fe1229c381", GitTreeState:"clean", BuildDate:"2017-08-17T17:03:51Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"", Minor:"", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2018-01-26T19:04:38Z", GoVersion:"go1.9.1", Compiler:"gc", Platform:"linux/amd64"}
```

**How to reproduce it** (as minimally and precisely as possible):
kubectl version
**Output of `minikube logs` (if applicable)**:

**Anything else do we need to know**:

**Comment by FlorianLehmann** [comment]
This problem seems to be related to #2505 
https://github.com/kubernetes/minikube/issues/2505

	// TODO: Deprecate gitMajor and gitMinor, use only gitVersion
	// instead. First step in deprecation, keep the fields but make
	// them irrelevant. (Next we'll take it out, which may muck with
	// scripts consuming the kubectl version output - but most of
	// these should be looking at gitVersion already anyways.)
	gitMajor string = "" // major version, always numeric
	gitMinor string = "" // minor version, numeric possibly followed by "+"

**Comment by afbjorklund** [comment]
This is a duplicate (of #2505), the problem has been fixed (in ab52ab1f81b2e7f72caa78b55047cbe018b6a33c) after `0.25.0` was released...

But there is now a need to *rebuild* the `localkube` binary, and re-upload it to the default GCS location:

* https://storage.googleapis.com/minikube/k8sReleases/v1.9.0/localkube-linux-amd64
* https://storage.googleapis.com/minikube/k8sReleases/v1.9.0/localkube-linux-amd64.sha256

Only project maintainers can do that.

@r2d4 ?

**Comment by jdxyw** [comment]
Any schedule to re-do it?

**Comment by cristian04** [comment]
+1

**Comment by afbjorklund** [comment]
The information is **also** missing from minikube v0.25.2 with localkube v1.9.4:
`Server Version: version.Info{Major:"", Minor:"", GitVersion:"v1.9.4", GitCommit:"bee2d1505c4fe820744d26d41ecd3fdd4a3d6546", GitTreeState:"clean", BuildDate:"2018-03-21T21:48:36Z", GoVersion:"go1.9.1", Compiler:"gc", Platform:"linux/amd64"}`

This because it was built from an older baseline 67a6b02320bf14757bcaa4396c4bf8dd01600e81, without the fix ab52ab1f81b2e7f72caa78b55047cbe018b6a33c

**Comment by jroucheton** [comment]
Any news for building a new release for fixing this issue?
